src/openforms/submissions/tests/test_submission_report.py

```diff
@@ -97,6 +99,35 @@ def test_report_generation(self, mock_request):
         # report.content.name contains the path too
         self.assertTrue(report.content.name.endswith("Test_Form.pdf"))
 
+    def test_report_is_generated_in_same_language_as_submission(self):
+        submission = SubmissionFactory.create(
+            language_code="en",
+            completed=True,
+            form__name_en="Translated form name",
+        )
+        FormStepFactory.create(
+            form=submission.form,
+            form_definition__name="Walsje",
```

Comment:
```suggestion
            form_definition__name_nl="Walsje",
```
makes it just that bit more explicit